### PR TITLE
docs: document LightAccount v2

### PR DESCRIPTION
**Pull request by dphilipson**
## PR-Codex overview
The focus of this PR is to introduce and enhance Multi-Owner Light Accounts in the Alchemy ecosystem.

### Detailed summary
- Added support for Multi-Owner Light Accounts in Alchemy
- Updated entry points for Multi-Owner Light Accounts
- Provided methods to manage ownership of Multi-Owner Light Accounts

> The following files were skipped due to too many changes: `site/smart-accounts/multi-owner-light-account/index.md`, `site/smart-accounts/light-account/index.md`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: site/smart-accounts/light-account/index.md [hunk 1]
```diff
@@ -33,18 +35,46 @@ The code snippet below demonstrates how to use Light Account with Account Kit. I
...
+
 ::: tip Address calculation
-For the Light Account, the address of the smart account will be calculated as a combination of [the owner and the salt](https://github.com/alchemyplatform/light-account/blob/main/src/LightAccountFactory.sol#L24-L33). You will get the same smart account address each time you supply the same `owner`, the signer used to create the account for the first time. Alternatively, you can supply `salt` if you want a different address for the same `owner` param (the default salt is `0n`).
+For the Light Account, the address of the smart account will be calculated as a combination of the version, [the owner, and the salt](https://github.com/alchemyplatform/light-account/blob/main/src/LightAccountFactory.sol#L24-L33). You will get the same smart account address each time you supply the same `version` and `owner`. Alternatively, you can supply `salt` if you want a different address for the same `version` and `owner` params (the default salt is `0n`).
```
Comment (jaypaik) [inline site/smart-accounts/light-account/index.md:45]: Nit: pin to v2.0.0 tag https://github.com/alchemyplatform/light-account/blob/v2.0.0/src/LightAccountFactory.sol#L24-L33

File: site/smart-accounts/light-account/index.md [hunk 2]
```diff
@@ -33,18 +35,46 @@ The code snippet below demonstrates how to use Light Account with Account Kit. I
...
 The following table lists the deployed factory and account implementation contract addresses for `LightAccount` on different chains:
 
+### v2.0.0
+
+| Chain            | Factory Address                            | Account Implementation                     |
+| ---------------- | ------------------------------------------ | ------------------------------------------ |
+| Eth Mainnet      | 0x0000000000400CdFef5E2714E63d8040b700BC24 | 0x8E8e658E22B12ada97B402fF0b044D6A325013C7 |
+| Eth Sepolia      | 0x0000000000400CdFef5E2714E63d8040b700BC24 | 0x8E8e658E22B12ada97B402fF0b044D6A325013C7 |
+| Polygon Mainnet  | 0x0000000000400CdFef5E2714E63d8040b700BC24 | 0x8E8e658E22B12ada97B402fF0b044D6A325013C7 |
+| Polygon Mumbai   | 0x0000000000400CdFef5E2714E63d8040b700BC24 | 0x8E8e658E22B12ada97B402fF0b044D6A325013C7 |
```
Comment (jaypaik) [inline site/smart-accounts/light-account/index.md:63]: We can take Mumbai out, it's not deployed there and the network is deprecated similar to Goerli.

File: site/smart-accounts/multi-owner-light-account/index.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+## [Deployment addresses](https://github.com/alchemyplatform/light-account/tree/v2.0.0/deployments)
+
+The following table lists the deployed factory and account implementation contract addresses for `MultiOwnerLightAccount` on different chains:
+
+| Chain            | Factory Address                            | Account Implementation                     |
+| ---------------- | ------------------------------------------ | ------------------------------------------ |
+| Eth Mainnet      | 0x000000000019d2Ee9F2729A65AfE20bb0020AefC | 0xd2c27F9eE8E4355f71915ffD5568cB3433b6823D |
+| Eth Sepolia      | 0x000000000019d2Ee9F2729A65AfE20bb0020AefC | 0xd2c27F9eE8E4355f71915ffD5568cB3433b6823D |
+| Polygon Mainnet  | 0x000000000019d2Ee9F2729A65AfE20bb0020AefC | 0xd2c27F9eE8E4355f71915ffD5568cB3433b6823D |
+| Polygon Mumbai   | 0x000000000019d2Ee9F2729A65AfE20bb0020AefC | 0xd2c27F9eE8E4355f71915ffD5568cB3433b6823D |
```
Comment (jaypaik) [inline site/smart-accounts/multi-owner-light-account/index.md:57]: Same!

**Review by jaypaik — DISMISSED** [review]
Looks good to me, with just a couple nits! Thanks for correcting existing spelling mistakes too.

**Comment by dphilipson** [comment]
Thanks @jaypaik! Updated based on your comments.

**Review by moldy530 — APPROVED** [review]

**Pull request merged**